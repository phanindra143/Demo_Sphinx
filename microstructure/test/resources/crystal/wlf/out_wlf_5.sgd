*if,ifprop,eq,2,then
  !c_ij(1,1)=     0.0000
  !c_ij(1,2)=     0.0000
  !c_ij(2,2)=     0.0000
  !c_ij(1,3)=     0.0000
  !c_ij(2,3)=     0.0000
  !c_ij(3,3)=     0.0000
  !c_ij(1,4)=     0.0000
  !c_ij(2,4)=     0.0000
  !c_ij(3,4)=     0.0000
  !c_ij(4,4)=     0.0000
  !c_ij(1,5)=     0.0000
  !c_ij(2,5)=     0.0000
  !c_ij(3,5)=     0.0000
  !c_ij(4,5)=     0.0000
  !c_ij(5,5)=     0.0000
  !c_ij(1,6)=     0.0000
  !c_ij(2,6)=     0.0000
  !c_ij(3,6)=     0.0000
  !c_ij(4,6)=     0.0000
  !c_ij(5,6)=     0.0000
  !c_ij(6,6)=     0.0000
*endif
*if,ifprop,eq,1,then
  !k_ij(1,1)=   157.3612
  !k_ij(1,2)=     0.0419
  !k_ij(2,2)=   157.9752
  !k_ij(1,3)=     0.2594
  !k_ij(2,3)=     0.1763
  !k_ij(3,3)=   157.8150
MP,Kxx,1,  157.36117
MP,Kyy,1,  157.97524
MP,Kzz,1,  157.81495
MP,Kxx,2,  157.97524
MP,Kyy,2,  157.36117
MP,Kzz,2,  157.81495
MP,Kxx,3,  157.97524
MP,Kyy,3,  157.36117
MP,Kzz,3,  157.81495
MP,Kxx,4,  157.36117
MP,Kyy,4,  157.97524
MP,Kzz,4,  157.81495
MP,Kxx,5,  157.36117
MP,Kyy,5,  157.97524
MP,Kzz,5,  157.81495
MP,Kxx,6,  157.97524
MP,Kyy,6,  157.36117
MP,Kzz,6,  157.81495
MP,Kxx,7,  157.97524
MP,Kyy,7,  157.36117
MP,Kzz,7,  157.81495
MP,Kxx,8,  157.36117
MP,Kyy,8,  157.97524
MP,Kzz,8,  157.81495
MP,Kxx,9,  157.81495
MP,Kyy,9,  157.36117
MP,Kzz,9,  157.97524
MP,Kxx,10,  157.36117
MP,Kyy,10,  157.81495
MP,Kzz,10,  157.97524
MP,Kxx,11,  157.36117
MP,Kyy,11,  157.81495
MP,Kzz,11,  157.97524
MP,Kxx,12,  157.81495
MP,Kyy,12,  157.36117
MP,Kzz,12,  157.97524
MP,Kxx,13,  157.81495
MP,Kyy,13,  157.36117
MP,Kzz,13,  157.97524
MP,Kxx,14,  157.36117
MP,Kyy,14,  157.81495
MP,Kzz,14,  157.97524
MP,Kxx,15,  157.36117
MP,Kyy,15,  157.81495
MP,Kzz,15,  157.97524
MP,Kxx,16,  157.81495
MP,Kyy,16,  157.36117
MP,Kzz,16,  157.97524
MP,Kxx,17,  157.97524
MP,Kyy,17,  157.81495
MP,Kzz,17,  157.36117
MP,Kxx,18,  157.81495
MP,Kyy,18,  157.97524
MP,Kzz,18,  157.36117
MP,Kxx,19,  157.81495
MP,Kyy,19,  157.97524
MP,Kzz,19,  157.36117
MP,Kxx,20,  157.97524
MP,Kyy,20,  157.81495
MP,Kzz,20,  157.36117
MP,Kxx,21,  157.97524
MP,Kyy,21,  157.81495
MP,Kzz,21,  157.36117
MP,Kxx,22,  157.81495
MP,Kyy,22,  157.97524
MP,Kzz,22,  157.36117
MP,Kxx,23,  157.81495
MP,Kyy,23,  157.97524
MP,Kzz,23,  157.36117
MP,Kxx,24,  157.97524
MP,Kyy,24,  157.81495
MP,Kzz,24,  157.36117
*endif
