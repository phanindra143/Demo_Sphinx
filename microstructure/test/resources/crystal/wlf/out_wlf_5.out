            ifx                 q_x                 q_y                 q_z     
  /% source: R:/166_320/Ablage_Simulationen/Modellbildung_Simulation_CMC-SiC_[Vorlauf+WiMi+MaVo]/04_04_2017_Modellierung_Matrix_mikro_ditt_[WiMi]/2_Waermeleitfähigket_SiC-Si\Rein_ohne_Poren\80-20
             3.00000      -2458785.08096          -736.09086        -14641.92048
  /%
            ifx                 dtx                 dty                 dtz     
  /%
             3.00000         15625.00000             0.62771            69.69197
  /%
            ifx                 q_x                 q_y                 q_z     
  /%
             2.00000           146.56066      -2468363.97024         -3644.87215
  /%
            ifx                 dtx                 dty                 dtz     
  /%
             2.00000            -5.29003         15625.00000             5.90864
  /%
            ifx                 q_x                 q_y                 q_z     
  /%
             1.00000        -12660.83657         -2378.21460      -2465873.49444
  /%
            ifx                 dtx                 dty                 dtz     
  /%
             1.00000            52.10490            -2.65494         15625.00000
  /%
